style.css

```diff
@@ -1,468 +1,1180 @@
+  padding-left: 57px;
+  padding-top: 28px;
+}
+
+.reting__text {
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 21px;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.text__link:nth-child(2) {
+  padding-top: 17px;
+  padding-bottom: 15px;
+}
+
+.text__link:nth-child(3) {
+  display: flex;
+  padding-bottom: 17px;
+}
+
+.text__link:nth-child(4) {
+  padding-bottom: 16px;
+}
+
+.star__link:nth-child(1) {
+  margin-top: -2px;
+}
+
+.star__link:nth-child(2) {
+  padding-top: 9px;
+  padding-bottom: 10px;
+}
+
+.star__link:nth-child(4) {
+  padding-top: 8px;
+  padding-bottom: 9px;
+}
+
+.text__link__title {
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 12px;
+  text-transform: uppercase;
+  padding-top: 7px;
+  padding-left: 2px;
+  color: #fff;
+}
+
+.reting__star {
+  margin-left: 32px;
+  width: 160px;
+  justify-content: space-between;
+}
+
+.star__foto {
+  width: 25px;
+}
+
+.star__foto:nth-child(2) {
+  margin-left: -1px;
+  margin-right: -2px;
+}
+
+.star__foto:nth-child(4) {
+  margin-right: -4px;
+}
+
+.star__list,
+.text__list {
+  list-style: none;
+}
+
+.information__block__aside-five {
+  padding-top: 5px;
+}
+
+.aside-second__five {
+  display: flex;
+  padding-left: 64px;
+  padding-top: 92px;
+}
+
+.text__foto {
+  width: 65px;
+  height: 82px;
+  padding-top: 10px;
+}
+
+.number__title__aside-five,
+.number__subtitle__aside-five {
+  font-size: 50px;
+}
+
+.number__title__aside-five {
+  padding-left: 23px;
+}
+
+.number__subtitle__aside-five {
+  padding-top: 0;
+  padding-left: 13px;
+}
+
+.number__text__aside-five {
+  font-size: 25px;
+  margin-top: -9px;
+  padding-left: 24px;
+}
+
+.group__second__aside-five {
+  justify-content: center;
+  padding-right: 27px;
+  padding-top: 44px;
+}
+
+.section-third__car {
+  padding-top: 172px;
+  padding-left: 344px;
+}
+
+.car__foto {
+  position: relative;
+  width: 100%;
+}
+
 /*Адаптивное отображение*/
+
+@media (max-width: 1411px) {
+  .section-second {
+    background-size: 100%;
+  }
+}
+
+@media (max-width: 1300px) {
+  .section-second {
+    background-position: -5px 123px;
+  }
+}
+
+@media (max-width: 1251px) {
+  .section-second {
+    background-position: -5px 200px;
+  }
+}
+
+@media (max-width: 1121px) {
+  .content__img {
+    padding-right: 0;
+  }
+
+  .section-second {
+    background-position: -5px 280px;
+  }
+}
+
 @media (max-width: 1024px) {
-    .header__menu,
-    .header__button {
-        display: none;
-    }
-    .header__burger {
-        padding-left: 518px;
-        padding-top: 24px;
-    }
-    /*.header__burger {*/
-    /*    width: 36px;*/
-    /*    height: 22px;*/
-    /*    padding-left: 417px;*/
-    /*}*/
+  .header {
+    width: 100%;
+  }
+
+  .header__menu,
+  .header__button,
+  .menu__link {
+    display: none;
+  }
+
+  .section-first {
+    background-position-y: 80px;
+  }
 
+  .information {
+    max-width: 700px;
+  }
+
+  .content__img {
+    width: 100%;
+  }
+
+  .aside {
+    height: 100%;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)